adminer-dg/Dockerfile

```diff
@@ -28,6 +29,8 @@ RUN echo '@testing http://nl.alpinelinux.org/alpine/edge/testing' >> /etc/apk/re
 WORKDIR /srv
 EXPOSE 80
 
+ENTRYPOINT ["dumb-init", "--"]
+
 CMD /usr/bin/php \
```

Comment:
No, you CAN'T combine exec form ENTRYPOINT with shell form CMD. The article explains why. Change this to exec form too.

Reply:
It works, but you're right, probably shouldn't.

Reply:
Ok, so calling it in shell form (current) is actually required here, because the arguments use environment variables and you need shell to replace them.

There is probably some way of handling this even in exec form, but I don't have time to handle this now.

Reply:
There isn't any good way actually. In this where the CMD is actually a command on it's own and not just params for ENTRYPOINT it should work this way although in reality it will execute like this:

```
dumb-init -- /bin/sh -c /usr/bin/php ...
```

But yeah, it works.

Reply:
Yes, while before it was executed like `/bin/sh -c /usr/bin/php ...`, so the dumb-init works in that it becomes PID 1 init process and nothing else is changed.

As far as I'm concerned it's good enough - it works and it keeps the dockerfiles simple.

Reply:
Yeah, agreed.